tests/services/inventory/cai_temporary_storage_test.py

```diff
@@ -0,0 +1,149 @@
+"""Tests for google3.experimental.users.ahoying.forseti-security.tests.services.inventory.cai_temporary_storage."""
```

Comment:
We might want to remove the ref to google3 here

Reply:
Oops!